### PR TITLE
Funding loans dashboard

### LINKED ISSUE #17: Students search page
Page to display ongoing loan requests

### PR DESCRIPTION
**Pull request by pferreira101**
Closes #17 : 

- Add /students page where all the loans on funding are listed through students cards
- Allows the user to do a filtered search (name, country and course) on funding loans

### CODE REVIEW

File: web/src/components/StudentCard.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    transform: "scale(1.1)",
+  },
+});
+
+const StyledCard = styled(Card)({
+  backgroundColor: "#FCFCFC",
+  borderRadius: 20,
+  "&:hover": {
+    boxShadow: "0px 10px 10px 0",
+  },
```
Comment (Bishop19) [inline web/src/components/StudentCard.js]: effect too big imo

File: web/src/containers/students/Students.js [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+            handleOptionClick={updateCourseFilter}
+          />
+        </Grid>
+      </SearchArea>
+      <Separator />
+      {filteredStudents.length > 1 ? (
+        <StudentsGrid
+          container
+          direction="row"
+          justify="center"
```
Comment (Bishop19) [inline web/src/containers/students/Students.js]: justify left (default) is better imo

**Review by Bishop19 — COMMENTED** [review]
Nice work!

File: web/src/components/StudentCard.js (hunk 1 above)
Comment (pferreira101) [reply web/src/components/StudentCard.js]: Changed to "0px 5px 5px 0px". Check again please

**Comment by pferreira101** [comment]
This page should be ready to be merged. All the features missing when this PR was opened are now implemented. There are still 2 missing features: progress bar on loan card is static (missing data from backend); card click doesn't work as intended because the loan page automatically redirects to the home page (fixed in another PR I guess...)

File: web/src/components/StudentCard.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            name={name}
+            course={course}
+            school={school}
+            tuition={tuition}
+            destination={destination}
+          />
+        </CardContent>
+      </StyledCard>
+      <Box style={{ width: "100%" }}>
+        <Progress completed={10} slim />
```
Comment (Bishop19) [inline web/src/components/StudentCard.js:86]: hardcoded?

**Review by Bishop19 — APPROVED** [review]

**Review by joaop21 — APPROVED** [review]
I think that you'll need to merge with main and resolve conflicts in student.js

File: web/src/components/StudentCard.js (hunk 3 above)
Comment (pferreira101) [reply web/src/components/StudentCard.js:86]: I opened #66 for this

**Comment by pferreira101** [comment]
> I think that you'll need to merge with main and resolve conflicts in student.js

Already did this, can you check if student.js is as intended? Got some conflicts as the formatter had not been run on previous commits

**Review by joaop21 — APPROVED** [review]

**Pull request merged**